src/services/api.js

```diff
@@ -379,6 +379,16 @@ class SDKWrapper {
       .catch(() => null);
   }
 
+  deleteUserAvatar(walletId: string): Promise<boolean> {
+    return Promise.resolve()
+      .then(() => this.pillarWalletSdk.user.deleteProfileImage({ walletId }))
+      .then(response => response.status === 204)
+      .catch((error) => {
+        reportLog('Failed to delete user avatar', { error }, Sentry.Severity.Error);
+        return false;
+      });
```

Comment:
Just a comment, if the return type of this function is already a Promise, do we need return Promise.resolve()? For example, could the function be written like this:

```suggestion
  deleteUserAvatar(walletId: string): Promise<boolean> {
    await this.pillarWalletSdk.user.deleteProfileImage({ walletId }))
      .then(response => response.status === 204)
      .catch((error) => {
        reportLog('Failed to delete user avatar', { error }, Sentry.Severity.Error);
        return false;
      };
```

Reply:
It doesn't really matter but it's a convention in this file. Each api function starts with Promise.resolve(). Maybe code aesthetic?

Reply:
No problem - not going to cry about this one 😂